cms/static/sass/views/_settings.scss

```diff
@@ -104,10 +109,11 @@
       margin-top: ($baseline/4);
       color: $gray-d1;
     }
-    .tip-inline{
-    display: inline;
-    margin-left: 5px;
-  }
+
+    .tip-inline {
+      display: inline;
+      margin-left: 5px;
```

Comment:
I know this is existing styling, but adding in RTL support @include margin-left($baseline/4); would be great!